desktop/cmp/dash/DashContainerModel.js

```diff
@@ -125,7 +125,15 @@ export class DashContainerModel {
         // Initialize GoldenLayout with initial state once ref is ready
         this.addReaction({
             when: () => this.containerRef.current,
-            run: () => this.loadStateAsync(initialState)
+            run: () => {
```

Comment:
Any reason not to just set state=initialState in the constructor, and then just have a single reaction?

Reply:
GREAT suggestion! Will make that change now